### DISCUSSION TITLE
How to use different templates depending on how you create a note?

### DISCUSSION THREAD

**Discussion by Fertion** [SilentVoid13/Templater#621] [Help] [answered]
Hello. I want that when clicking on a link leading to a note that has not yet been created, one template is used, and when a note is normally created, another template is used.

How to implement it?

**Comment by liamcain** [comment]
Someone asked a similar question on Discord. One way would be to check if the new file name starts with "Untitled" https://discord.com/channels/686053708261228577/707816848615407697/964246562563833917

**Reply by Fertion** [reply]
> Someone asked a similar question on Discord. One way would be to check if the new file name starts with "Untitled" https://discord.com/channels/686053708261228577/707816848615407697/964246562563833917

Thanks for the tip.

I looked here https://github.com/SilentVoid13/Templater/discussions/259 and I'm trying to write code, but I don't understand programming at all, what did I do wrong?


<%*
let title = tp.file.title
var TemplateText;
if(title.startsWith("Untitled")) { 
TemplateText= "Test 1" ; 
} else {
TemplateText= "Test 2" ; 
};
%>

<%*  `${TemplateText}` %>

**Reply by AB1908** [reply]
```js
<%*
let title = tp.file.title
var TemplateText;
if(title.startsWith("Untitled")) {
    TemplateText= "Test 1" ;
} else {
    TemplateText= "Test 2" ;
};

tR += TemplateText;
%>
```

**Reply by Fertion** [reply]
Excellent thank you!

And the last thing I can't figure out is how can I now display such a template instead of Test 2?

I thought to enter it as a variable, but it doesn't work like that, as I now understand.

```
<< [yesterday](<% tp.date.now("YYYY.MM.DD", -1, tp.file.title, "YYYY.MM.DD") %>) | [tomorrow](<% tp.date.now("YYYY.MM.DD", 1, tp.file.title, "YYYY.MM.DD") %>) >>
[[Main MOC]]

---
# Plan for the day
- [ ] 9:00 [[Morning ritual]]
- [ ]
```

**Reply by AB1908** [reply]
Not sure what you mean by "display template" here. Your code block doesn't seem to be particularly related either.

**Comment by Fertion** [comment, marked as answer]
Thanks to AB1908 for helping out on discord.
Final code that does exactly what I need:

```
<%*
let title = tp.file.title
var TemplateText;
if(title.startsWith("Untitled")) { 
	TemplateText = "Test 1" ; 
} else {
	TemplateText = await tp.file.include("[[TestTemplate]]")
};

tR += TemplateText;
%>
```
